### ISSUE TITLE
selective method mirroring

### ISSUE THREAD

**Issue by nikomatsakis** [duchess-rs/duchess#4]
Right now we only support mirroring *all* the methods in a class:

```rust
duchess::java_package! {
    package foo;
    class Bar { * }
}
```

but we should support selective mirroring. What's a bit unclear is *how* to do that. By name? By full signature (to account for overloaded methods)? I'm inclined to just go with "by name" for simplicity, and because I imagine we'll want to support it for ergonomics anyway, maybe something like this?

```rust
duchess::java_package! {
    package foo;
    class Bar { Bar(), toString(), computeName() }
}
```

where `Bar()` refers to the constructor(s).

That would give us space to adopt full signatures in the future if that ever seemed necessary.

**Comment by nikomatsakis** [comment]
### Brief mentoring instructions

Extend the [`MemberListing`](https://github.com/nikomatsakis/duchess/blob/0a1dd1ec20bbe8aee52077ef149f6019a99a5108/macro/src/argument.rs#L91-L93) enum with a new option, `MethodNamed(String)` or something like that, and extend the [parser code for that enum](https://github.com/nikomatsakis/duchess/blob/0a1dd1ec20bbe8aee52077ef149f6019a99a5108/macro/src/argument.rs#L96-L102) to consume either a `*` (as today) or an [identifier](https://github.com/nikomatsakis/duchess/blob/main/macro/src/parse.rs#L104) followed by an (empty) [parentheses-delimited tokentree](https://github.com/nikomatsakis/duchess/blob/main/macro/src/parse.rs#L75). 

Add that list of methods to the `SpannedClassInfo` struct and modify the [`SpannedClassInfo` codegen](https://github.com/nikomatsakis/duchess/blob/main/macro/src/codegen.rs#L21) to [only translate *some* of the methods into Rust code](https://github.com/nikomatsakis/duchess/blob/0a1dd1ec20bbe8aee52077ef149f6019a99a5108/macro/src/codegen.rs#L47).

**Comment by nikomatsakis** [comment]
Implemented in #25
